# Boat pose in local ENU map frame (meters, radians)
# Heading: 0 = East, positive = counter-clockwise from above (right-handed ENU)
std_msgs/Header header
float64 east
float64 north
float64 heading_rad
